scripts/ssm-build-deb.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Execute the following as root to install lintian and fpm
+# apt-get install lintian
+# apt-get install ruby ruby-dev rubygems build-essential
+# gem install --no-ri --no-rdoc fpm
+
+# Then run this file, altering the version.
+# This file will create two versions of the deb file:
+# - apel-ssm_<version>_all.deb contains all the files necessary to run a
+#   the SSM as a sender.
+# - apel-ssm-service_<version>_amd64.deb will install service files
+#   necessary to run the SSM as a receiver on amd64 style architectures.
+# After building apel-ssm_<version>_all.deb, this script will run it
+# against lintian to highlight potential issues to the builder.
+
+set -eu
+
+TAG=2.2.0-1
+
+SOURCE_DIR=~/debbuild/source
+BUILD_DIR=~/debbuild/build
+
+# Where to install the python lib files
+PYTHON_INSTALL_LIB=/usr/lib/python2.7/dist-packages
+
+# Split the tag into version and package number
+# so they can be passed to fpm separately.
+# This will work with RCs of the form X.Y.Z-0.A.rcA
```

Comment:
And any other package that uses our version scheme? e.g 1.2.3-1.1.alpha1

Reply:
it should do, I think it will split on the -. If it does I'll clarify the comment.

Reply:
clarified